packages/cli/src/lib/implementation/utils.ts

```diff
@@ -1,5 +1,7 @@
 // log error and flush stdout so that Yargs doesn't suppress it
 // related issue: https://github.com/yargs/yargs/issues/2118
+import { toArray } from '@code-pushup/utils';
+
 export function logErrorBeforeThrow<
```

Comment:
The comment belongs to the `logErrorBeforeThrow` function, not the import.

```suggestion
import { toArray } from '@code-pushup/utils';

// log error and flush stdout so that Yargs doesn't suppress it
// related issue: https://github.com/yargs/yargs/issues/2118
export function logErrorBeforeThrow<
```

Reply:
You are right